app/views/layouts/_institute_footer.html.erb

```diff
@@ -12,6 +12,7 @@
       </div>
       <div class="license">Licensed under the <a href="https://creativecommons.org/licenses/by-nc/2.0/" class="license-cc">Creative Commons Attribution Non-Commercial License</a> unless otherwise noted. <a href="https://libraries.mit.edu/research-support/notices/copyright-notify/">Notify us about copyright concerns</a>.
       </div><!-- end .footer-info-institute -->
+      <div>Access level: <%= access_level %></div>
```

Comment:
Why is this here? What's the rationale for displaying it to users, and has UX seen this?